libs/commands/publish/src/lib/publish-command.spec.ts

```diff
@@ -31,12 +31,12 @@ jest.mock("./get-unpublished-packages", () => ({
 jest.mock("./git-checkout");
 
 // lerna version mocks
-jest.mock("@lerna/commands/version/lib/git-push");
-jest.mock("@lerna/commands/version/lib/is-anything-committed", () => ({
+jest.mock("../../../version/src/lib/git-push");
```

Comment:
I suspect this will still work as `"@lerna/commands/version/src/lib/git-push"`. Can you please try that for all these imports of version command helpers? It's better if we can keep this as an absolute import since it crosses library boundaries.

This way, the publish tests won't start failing if we ever change the directory structure around the version command.

Reply:
Thx for the review. This was actually the fix for the failing tests. Using the pathmap here is not working then the mock will not be applied and the tests fail. I also tried to refactor the git-push into a real typescript module but it was still not working.
 I fully understand your point. A quickfix would be the proposed solution for a cleaner one I have to investigate a little more.

Reply:
Uploaded a solution keeping original paths. So what I did is considering the path config from tsconfig. Maybe we could do this on top-level in jest.preset.js? 
I think this should also work but I did not do it, because its only needed by publish tests.